src/api/spec/models/user_spec.rb

```diff
@@ -288,4 +288,49 @@
       end
     end
   end
+
+  describe '#declined requests' do
+    let(:target_package) { create(:package) }
+    let(:source_package) { create(:package) }
+    let(:confirmed_user) { create(:confirmed_user, login: 'confirmed_user') }
+    let(:new_bs_request) { create(:bs_request, creator: confirmed_user) }
+    let(:declined_bs_request) {
+      create(:declined_bs_request,
+             target_project: target_package.project,
+             target_package: target_package,
+             source_project: source_package.project,
+             source_package: source_package,
+             creator: confirmed_user)
+    }
+    let(:admin_bs_request) {
+      create(:declined_bs_request,
+             target_project: target_package.project,
+             target_package: target_package,
+             source_project: source_package.project,
+             source_package: source_package,
+             creator: admin_user)
+    }
+
+    let(:subject) { confirmed_user.declined_requests }
```

Comment:
Seems the "right" way would be to wrap it like this: [subject { ... }](https://relishapp.com/rspec/rspec-core/v/3-5/docs/subject/explicit-subject)